src/pages/Checkout/CheckoutPage.vue

```diff
@@ -850,7 +850,8 @@ export default {
 				this.setUpdatingTotals(false);
 			});
 		},
-		completeTransaction(transactionId) {
+		completeTransaction(payload) {
+			const transactionId = typeof payload === 'object' ? payload.transactionId : payload;
```

Comment:
Are there situations where this won't be an `object` now?

Reply:
Yep, here: https://github.com/kiva/ui/blob/1766cb68be5e5105e50802dde6a177a6e6dd01e2/src/pages/Checkout/CheckoutPage.vue#L100